### ISSUE TITLE
[M] Decommission old/out of date urls for Day Use Pass

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/parks-reso-admin#390]
#### Description:

There are new URLs we should be using for testing, etc. It is confusing and folks (ah hem...the PO ;0 ) who keeps testing on the old urls and getting very confused. 

The URLs we should be using are:
Test DUP-Public : [test-dup.bcparks.ca](https://test-dup.bcparks.ca/) (d3ot7vkkckpzq8.cloudfront.net)
Test DUP-Admin: [test-dup-admin.bcparks.ca](https://test-dup-admin.bcparks.ca/) (d1du76e5swm15v.cloudfront.net)
Dev DUP-Public :https://dev-dup.bcparks.ca/
Dev DUP-Admin : https://dev-dup-admin.bcparks.ca/

#### Development Checklist:

- [x] Remove/retire/decommission old urls that are no longer needed
- [ ]Determine if a redirect message is needed to point people to the correct urls. 
- [x] Old links (TBC)
    - https://d3jkh5unv9de6y.cloudfront.net/dayuse/login
    - https://d2yc82svj0l8d5.cloudfront.net/dayuse/ 
    - Dev DUP-Public : https://dn7im283dz4ia.cloudfront.net/dayuse/
    - Dev DUP-Admin : https://d2ya06v0khe71p.cloudfront.net/
- [x] Carefully identify the resources to be abandoned

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
Once manuji adds additional links to decommission, setup for estimates.

**Comment by Dianadec** [comment]
@davidclaveau and/or @Christopher-walsh22 to add more info around the post-sam cleanup cloudfront things.

**Comment by manuji** [comment]
@Dianadec , the urls are now updated.

**Comment by Dianadec** [comment]
@manuji could you clarify what you mean by being updated? Where?

**Comment by manuji** [comment]
I updated the URLS we need to decommission in the ticket description. This ticket should be ready for implementation. @Dianadec

**Comment by davidclaveau** [comment]
Hey @manuji! 

This review should be pretty easy as it's just a smoke test to make sure TEST is still working as before 😄

I have removed the old links and resources in DEV and TEST for DUP. I've done my own testing and things are looking good and I haven't encountered any issues, but I just want to make sure I get a second pair of eyes on TEST before I move onto PROD and delete the old resources as well.

All the old resources are from the pre-SAM days. Here's what I deleted in case that's interesting to note:

- [x] Remove several old S3 buckets (old builds)
- [x] Remove old Cloudfront Distributions (old URLs - this doesn't apply to PROD)
- [x] Remove old DynamoDB tables
  - [x] parksreso
  - [x] parksreso-meta
  - [x] parksreso-metrics
- [x] Remove old API Gateway
- [x] Remove old Lambda Functions
- [x] Remove old SQS processes

Thanks! And let me know if you have any questions!

**Comment by manuji** [comment]
@davidclaveau , not sure of it's related to this ticket, but when I deployed DUP-Public to test, the build failed in the step where it executed a step in CloudFront. Refer to the failed action here: https://github.com/bcgov/parks-reso-public/actions/runs/13206803577/job/36871980918

**Comment by manuji** [comment]
Tested on TEST:
* Verified that the old db tables have been removed in DynamoDB.
*

**Comment by davidclaveau** [comment]
@manuji this should be resolved now! Ended up just being an old GH secret that was being pulled in during the job. I've gone ahead and fixed this for the other environments as well.

**Comment by manuji** [comment]
Tested on TEST & Dev: **Passed**
* Smoke test passed on DUP Public and Admin. 
* Passing this to PO review.

**Comment by LindsayMacfarlane** [comment]
Thanks @davidclaveau and @manuji! I was still able to access the second url (see below) when testing. Can you please confirm. 

Old links to be removed:

- https://d3jkh5unv9de6y.cloudfront.net/dayuse/login > No longer available
- https://d2yc82svj0l8d5.cloudfront.net/dayuse/ > **STILL AVAILABLE**
- Dev DUP-Public : https://dn7im283dz4ia.cloudfront.net/dayuse/ > No longer available
- Dev DUP-Admin : https://d2ya06v0khe71p.cloudfront.net/ > No longer available

**Comment by davidclaveau** [comment]
@LindsayMacfarlane I double checked and the link has been deleted in AWS 🤔 

The page is likely loading based on some cache in your browser - it was actually it was still loading for me until I hit hard-refresh a couple more times. Doing a cache-clear should correct that issue, but you can also test the link in an incognito browser and it will show the `This site can't be reached` screen as expected.

**Comment by LindsayMacfarlane** [comment]
Thanks for that clarification @davidclaveau! Approving and closing ticket :)
